packages/extension-bubble-menu/package.json

```diff
@@ -33,13 +33,14 @@
     "directory": "packages/extension-bubble-menu"
   },
   "sideEffects": false,
+  "dependencies": {
```

Comment:
Is there a reason why we include floating-ui in the deps now instead of letting users manage it themselves?

Reply:
My bad, I think I was running into dep resolution problems & this is something I tried that did nothing. Updating.

Though technically, this is mostly equivalent, since the dep is declared as a range.